gdpr/vendorlist-fetching_test.go

```diff
@@ -196,15 +196,15 @@ func TestDefaultVendorListPassthrough(t *testing.T) {
 	fetcher := newVendorListFetcher(context.Background(), testcfg, server.Client(), testURLMaker(server), 1)
 	list, err := fetcher(context.Background(), 2)
 	assert.NoError(t, err, "Error with fetching existing vendorlist: %v", err)
-	assert.Equal(t, uint16(2), list.Version(), "Expected to fetch mock list version 2, got version %d", list.Version())
+	assert.Equal(t, uint16(2), list.Version(), "Expected to fetch mock list version 1, got version %d", list.Version())
```

Comment:
Shouldn't the expected value be `uint16(1)` instead?

Reply:
No,. It should be version 2. I forgot to revert my change to the comment.

Reply:
Fixed.